<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EasyIni</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>QSettings</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="qtEdit">
        <property name="text">
         <string>qsettings.ini</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="qtWrite">
        <property name="text">
         <string>write</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="qtRead">
        <property name="text">
         <string>read</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>CustomSettings</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="customEdit">
        <property name="text">
         <string>custom_utf16le.ini</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="customWrite">
        <property name="text">
         <string>write</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="customRead">
        <property name="text">
         <string>read</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>EasyIni</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="easyEdit">
        <property name="text">
         <string>easyini.ini</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="easyWrite">
        <property name="text">
         <string>write</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="easyRead">
        <property name="text">
         <string>read</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>180</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
